src/assets/scripts/game/GameController.js

```diff
@@ -68,107 +71,56 @@ export default class GameController {
      */
     init_pre() {
         prop.game = game;
-        // TODO: with the assignment of `prop.game = game`, the rest of these assignments are redundant and can
-        // be removed after proper testing
-        prop.game.paused = true;
-        prop.game.focused = true;
-        prop.game.speedup = 1;
-        prop.game.frequency = 1;
-        prop.game.time = 0;
-        prop.game.delta = 0;
-        prop.game.timeouts = [];
 
+        // Set blurring function
         $(window).blur(() => {
             prop.game.focused = false;
         });
 
+        // Set un-blurring function
         $(window).focus(() => {
             prop.game.focused = true;
         });
 
-        prop.game.last_score = 0;
-        prop.game.score = {
-            arrival: 0,
-            departure: 0,
-
-            windy_landing: 0,
-            windy_takeoff: 0,
-
-            failed_arrival: 0,
-            failed_departure: 0,
-
-            warning: 0,
-            hit: 0,
-
-            abort: {
-                landing: 0,
-                taxi: 0
-            },
-
-            violation: 0,
-            restrictions: 0
-        };
-
-        prop.game.option = new GameOptions();
+        // Initialize event count
+        _forEach(GAME_EVENTS, (v, k) => {
+            prop.game.events[k] = 0;
+        });
```

Comment:
be sure to spell these two values out and give them contextual meaning.  what to `v` and `k` mean?
```javascript
_forEach(GAME_EVENTS, (gameEvent, key) => {
    prop.game.events[key] = 0;
});
```

in this case its rather subtle, but it makes it way easier for other developers to know whats going on without having to dig too far.  Sometimes it is obvious, like `i` in a for loop.  Other times, though, it is not.